grafana-plugin/src/components/Avatar/Avatar.styles.ts

```diff
@@ -0,0 +1,32 @@
+import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+
+export const getAvatarStyles = (_theme: GrafanaTheme2) => {
```

Comment:
I guess we don't need those unused `_theme` params?

Reply:
I'm ok with either way, up to you and Maxim

Reply:
> I'm ok with either way
I think so too, there is no strong opinion from my side